apps/admin/app/containers/BrokerCAD/VerifyAccount/index.tsx

```diff
@@ -62,8 +63,8 @@ class VerifyAccount extends React.PureComponent<Props, typeof initialState> {
 		const address = NormalizeAddress(account);
 		const signature = await signer.signMessage(address)
 
-		await SetUserVerified(signature, address, Timestamp.now());
-		await CreateReferrer(signature, address);
+		await setUserVerified(signature, address, DateTime.now());
+		await createReferrer(signature, address);
 		alert('Done');
```

Comment:
Alert?

Reply:
Yeah... it's Admin (internal only).

Still should be better, but...